README.md

```diff
@@ -75,9 +88,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. external css page that is linked with the html page with the code: <link href='css_file_name' rel="stylesheet"> (most specific)
+2. css code on the html page withing the <head></head> section using the tag <style></style>
+3. css elements in the html tags using "style= ..." inside the html tags on the code. (least specific)
 ```
```

Comment:
While external stylesheets are the preferred way of writing out CSS, the `style=...` method has the highest degree of specificity (i.e., if inline styling conflicts with something in an external stylesheet, it will win out over the stylesheet).